e2etest/bgp_test.go

```diff
@@ -1151,8 +1152,82 @@ var _ = ginkgo.Describe("BGP", func() {
 		table.Entry("IPV6", "fc00:f853:0ccd:e799::/116", "ipv6", "fc00:f853:ccd:e800::1/128"),
 	)
 
+	ginkgo.Context("validate FRR running configuration", func() {
+		ginkgo.It("Full BFD profile", func() {
+			bfdProfile := config.BfdProfile{
+				Name:             "fullBFDProfile1",
+				ReceiveInterval:  uint32Ptr(93),
+				TransmitInterval: uint32Ptr(95),
+				EchoInterval:     uint32Ptr(97),
+				EchoMode:         boolPtr(true),
+				PassiveMode:      boolPtr(true),
+				MinimumTTL:       uint32Ptr(253),
+			}
+
+			configData := config.File{
+				Pools: []config.AddressPool{
+					{
+						Name:      "bgp-test",
+						Protocol:  config.BGP,
+						Addresses: []string{v4PoolAddresses},
+					},
+				},
+				Peers:       peersForContainers(frrContainers, "ipv4"),
+				BFDProfiles: []config.BfdProfile{bfdProfile},
+			}
+
+			configData.Peers = withBFD(configData.Peers, bfdProfile.Name)
+
+			for i := range configData.Peers {
+				configData.Peers[i].KeepaliveTime = "13s"
+				configData.Peers[i].HoldTime = "57s"
+			}
+
+			err := configUpdater.Update(configData)
+			framework.ExpectNoError(err)
+
+			speakerPods := getSpeakerPods(cs)
+			for _, pod := range speakerPods {
+				podExecutor := executor.ForPod(pod.Namespace, pod.Name, "frr")
+
+				Eventually(func() string {
+					// We need to assert against the output of the command as a bare string, as
+					// there is no json version of the command.
+					cfgStr, err := podExecutor.Exec("vtysh", "-c", "show running-config")
+					framework.ExpectNoError(err)
```

Comment:
Can you return the err instead of failing the test? That way we won't fail if for some reason the Exec failed that would have worked given another iteration of the Eventually.
```
if err != nil {
	return err.Error()
}
```

Reply:
You are right! Going to fix it